product-scenarios/9-access-delegation/9.1-access-delegation-with-end-user-authorization/9.1.10-self-contained-access-token-as-authorization-grant/src/test/java/org/wso2/identity/scenarios/access/delegation/oauth2/selfcontainedtoken/OAuth2SelfContainedTokenImplicitGrantTest.java

```diff
@@ -140,7 +140,7 @@ public void atEnd() throws Exception {
     public void intiAuthorizeRequest() throws Exception {
 
         HttpResponse response = oAuth2CommonClient
-                .sendAuthorizeGet(clientId, null, redirectUri, OAuth2Constants.ResponseTypes.TOKEN, null);
+                .sendAuthorizeGet(clientId, "openid profile", redirectUri, OAuth2Constants.ResponseTypes.TOKEN, null);
```

Comment:
use constant `"openid profile"`

Reply:
Fixed with https://github.com/wso2/product-is/pull/14317/commits/1b4ab737ecfa0d7092d2c697a13b8ad39d9ad538